spec/support/helpers/api_helper.rb

```diff
@@ -50,6 +50,11 @@ def stub_offender(offender)
     stub_request(:post, T3_LATEST_MOVE_URL).with(
       body: [offender_no].to_json).
       to_return(body: [].to_json)
+
+    allow(ComplexityMicroService).
+      to receive(:get_complexity).
+        with(offender_no).
+        and_return(offender.fetch(:complexityLevel))
```

Comment:
Is there any reason for using this RSpec mock over a WebMock mock?

Doesn't necessarily need to be changed – just interested in the reason.

Reply:
It seemed easier (c.f. Community API where we do the same thing)